prov/efa/src/rdm/rxr_env.c

```diff
@@ -271,6 +291,9 @@ int rxr_env_get_use_device_rdma()
 {
 	int ret, use_device_rdma;
 
+	/* Get the default value of using device's RDMA capability */
+	use_device_rdma = get_use_device_rdma_default();
```

Comment:
Since you are here, please fix the comment of this function. I currently says:

```
 * @return		0 - If FI_EFA_USE_DEVICE_RDMA is not set
 * 				1 - If FI_EFA_USE_DEVICE_RDMA is set
 ```
which is not right.   this function return 0, when FI_EFA_USE_DEVICE_RDMA is set to 0/false/off.